### PR TITLE
[15 min fix] Fix timestamp error in Preact story cards

### PR DESCRIPTION
**Pull request by aitchiss**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Due to a typo in the Preact `Meta` component, we were rendering `<time>` elements without the relevant `dateTime` attribute, wherever we used the Preact-generated feed cards (e.g. the logged in home feed).

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

- Load the app locally and visit the home feed
- Using the dev tools inspector, look at the published date for a story card:

example of a story card with the date highlighted

- Check the HTML element has the `dateTime` attribute as expected, e.g:

screenshot of devtools inspector with dateTime highlighted

### UI accessibility concerns?

N/A

## Added/updated tests?

- [X] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [X] This change does not need to be communicated, and this is why not: very minor, non-user-facing fix

### CODE REVIEW

File: app/javascript/articles/components/Meta.jsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ export const Meta = ({ article, organization }) => {
         <a href={article.path} className="crayons-story__tertiary fs-xs">
           <PublishDate
             readablePublishDate={article.readable_publish_date}
-            publishedTimestap={article.published_timestamp}
+            publishedTimestamp={article.published_timestamp}
```
Comment (nickytonline) [inline app/javascript/articles/components/Meta.jsx:83]: I wonder why the proptypes didn't complain about this. 🤔

**Review by nickytonline — APPROVED** [review]
Good catch!

**Review by djuber — APPROVED** [review]

File: app/javascript/articles/components/Meta.jsx (hunk 1 above)
Comment (aitchiss) [reply app/javascript/articles/components/Meta.jsx:83]: I don't think it gives warnings if you pass an 'extra' prop... and the `publishedTimestamp` prop isn't marked as required (maybe it should be though?)

Comment (nickytonline) [reply app/javascript/articles/components/Meta.jsx:83]: Ahh didn't realize it was an extra prop. If it's always there, I'd add it as a required prop.

**Pull request merged**